test/shared_utilities/drivers.jl

```diff
@@ -81,39 +81,39 @@ end
     pa = ClimaLSM.PrescribedAtmosphere(f, f, f, f, f, f, f, FT(1);)
     pr = ClimaLSM.PrescribedRadiativeFluxes(FT, f, f, f)
     coords = (; surface = [1])
-    drivers = ClimaLSM.initialize_drivers(nothing, nothing, coords)
```

Comment:
changes in this file reflect that `update_drivers!` takes `p` now instead of `p.drivers`